docs/pages/blog/2019.md

```diff
@@ -0,0 +1,102 @@
+- Bootstrap has successfully released [a premium store](https://themes.getbootstrap.com/).
+The approach opens an opportunity to capture a fraction of the value Material-UI creates for its users and fuel it back into R&D on the framework.
+- The market for paid UI components is in the order of a couple of $100m/year.
+They are dozens of companies positioned in this market.
+While React is only one technology to build UI among others (jQuery, Angular, ASP.NET, Blazor, Vue, WPF, UWP, etc), we believe that Web and React will become the dominant technology in the next 5 years for enterprises. Material-UI is uniquely positioned to address this market with non-MIT features.
+- Building UIs should be simpler, it still too slow and complex.
+Designers and developers should benefit from a more integrated experience.
+- Every now and then, we witness the appearance of a new React UI component library built from scratch ([UXPin](https://adele.uxpin.com/) keeps track of some of them).
+And every time we asked ourselves, What could have we done differently to empower this library?
+We believe that starting from scratch, while maximizing freedom, is incredibly inefficient.
+Most of UI libraries out there need the same features but are implemented with a wide spectrum of accessibility, API and overall design quality.
+We won't rest until we successfully unify these efforts. It's a long term mission and will probably take years. This will likely mean releasing an unstyled version of our components.
+
+## Achievements
+
+- We have released [v4](/blog/material-ui-v4-is-out/).
+- We have introduced many new components (some in the core, some in the lab):
+  - [Autocomplete](/components/autocomplete/)
+  - [Backdrop](/components/backdrop/)
+  - [Breadcrumbs](/components/breadcrumbs/)
+  - [ButtonGroup](/components/button-group/)
+  - [Container](/components/container/)
+  - [Link](/components/links/)
+  - [Rating](/components/rating/)
+  - [Skeleton](/components/skeleton/)
+  - [Slider](/components/slider/)
+  - [TextareaAutosize](/components/textarea-autosize/)
+  - [TreeView](/components/tree-view/)
+- We have fixed a significant number of [accessibility issues](https://github.com/mui-org/material-ui/issues?q=is%3Aissue+label%3Aaccessibility+is%3Aclosed).
+- We have introduced global class names.
+- We have migrated the whole codebase to hooks.
+- We have migrated all the demos to TypeScript (source of truth).
+- We have introduced [native tree-shaking](/guides/minimizing-bundle-size/) support.
+- We have introduced [built-in localization](/guides/localization/).
+- We have remove a good chunk of external dependencies and increased the `features/bundle size` density.
+- We have introduced an [icon search page](/components/material-icons/).
+- We have released a [store for Material-UI](https://themes.material-ui.com/).
```

Comment:
Store for Material-UI themes?

Or 

theme store for Material-UI

Reply:
I think that "theme" is too restrictive, developers seems to value templates and design assets as much as themes.